com.github.petebuffon.yafc.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.petebuffon.yafc</id>
```

Comment:
Any reason you're using your username here?

Reply:
The upstream developer doesn't care about Linux support.